test/dynamo/test_misc.py

```diff
@@ -3055,6 +3055,76 @@ def f(pred, x):
         b = opt_fn(torch.tensor(True), torch.tensor([0.25, 0.25]))
         self.assertTrue(same(torch.sin(torch.tensor([0.25, 0.25])), b))
 
+    def test_nonzero_static(self):
```

Comment:
Is the out variant accessible from users? I tried call `torch.nonzero_static_out()` in the test, it said this can not be found. I think out variant can be accessed as function. No?